src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -201,6 +217,20 @@ private void handleJobElement(ImmutableList.Builder<KeyValueLabelMetadataEntity>
         jobHandled = true;
     }
 
+    private void handleDescriptionElement(ImmutableList.Builder<KeyValueLabelMetadataEntity> keyValueMapBuilder,
+            XMLStreamReader xmlStreamReader) {
+
+        String descriptionContent = "";
+        try {
+            descriptionContent = xmlStreamReader.getElementText(); //returns the text content of CDATA for description tag in our case
+        } catch (XMLStreamException e) {
+            log.error("Unable to parse the workflow description", e);
+            throw new RuntimeException("Unable to parse the workflow description", e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)